DoorShuffle.py

```diff
@@ -1182,8 +1184,8 @@ def reassign_key_doors(builder, world, player):
                         dp.pair = False
                 if not found:
                     world.paired_doors[player].append(PairedDoor(d1.name, d2.name))
-                    change_door_to_small_key(d1, world, player)
-                    change_door_to_small_key(d2, world, player)
+                change_door_to_small_key(d1, world, player)
```

Comment:
Not sure about this change. There are other doors in the paired_doors list besides key doors.

Reply:
True, but I figured that if something is chosen by the logic to be a key door, then it should be a key door, even if it was originally a different type of door. It's not an important change though

Reply:
Hmm.  It looks like this would happen if you have a different type of paired door, and the key layout decided to change them into a paired key door, so I think this is right.

Reply:
I've let this sit long enough, I'm incorporating this change into unstable for now.